src/vue/pages/SaleDetails.vue

```diff
@@ -161,5 +161,6 @@ export default {
   margin-top: .4rem;
   font-size: 1.6rem;
   color: $col-sale-details-subtitle;
+  word-wrap: break-word;
```

Comment:
Not such good property. Will bre
ak the words like this

Reply:
Discussed IRL